environment.yml

```diff
@@ -1,16 +1,25 @@
-name: tjpcov  # minimum environment for tjpcov development
+name: tjpcov
 channels: 
   - conda-forge
-  - defaults
 dependencies:
   - python>=3.8
-  - mpi4py
   - pip
-  - numpy
   - wheel
+  - black
+  - flake8
+  - pytest
+  - pytest-rerunfailures
+  - coveralls
+  - numpy
+  - scipy
+  - pyyaml
+  - pyccl >= 2.5.0, < 2.8.0
+  - sacc>=0.8
   - namaster
-  # tjpcov is only available at the moment through PyPi
-  - pip:
-    # Installing TJPcov in two steps to avoid making NaMaster installation to
-    # fail due to (https://github.com/LSSTDESC/NaMaster/issues/138)
-    - tjpcov[full]
```

Comment:
hmm, so this file will set up the environment but will not install tjpcov?

Reply:
Yes, I think the added step of installing TJPCov for development with `pip install -e .` is just one extra command and makes the development setup much cleaner.